raiden-dapp/src/components/overlays/InfoOverlay.vue

```diff
@@ -38,60 +40,77 @@ export default class InfoOverlay extends Vue {
 </script>
 
 <style lang="scss" scoped>
+@import '@/scss/dimensions';
 @import '@/scss/mixins';
 @import '@/scss/colors';
 
 .info-overlay {
-  align-items: center;
   background-color: $transfer-screen-bg-color;
   border-radius: 10px;
-  display: flex;
-  flex-direction: column;
-  height: calc(844px);
-  padding: 0 46px;
+  height: 844px;
   position: absolute;
   width: 100%;
   z-index: 102;
+
   @include respond-to(handhelds) {
     border-radius: 0;
     height: 100vh;
+    overflow-y: auto;
   }
 
-  &__header {
+  &__content {
+    align-items: center;
     display: flex;
-    height: 80px;
-    width: 100%;
-  }
+    flex-direction: column;
+    height: 100%;
+    @include respond-to(handhelds) {
+      margin-top: $ios-safe-area;
+    }
 
-  &__close-icon {
-    width: 100%;
-    display: flex;
-    justify-content: flex-end;
-  }
+    &__header {
+      display: flex;
+      height: 80px;
+      width: 100%;
+      @include respond-to(handhelds) {
+        height: 60px;
+      }
+    }
 
-  &__image {
-    margin-top: 46px;
-    height: auto;
-    width: 300px;
-  }
+    &__close-icon {
+      display: flex;
+      justify-content: flex-end;
+      padding-right: 36px;
+      width: 100%;
+    }
 
-  &__title {
-    color: $color-gray;
-    font-size: 24px;
-    margin: 36px 0;
-  }
+    &__image {
+      padding-top: 46px;
+      height: auto;
+      width: 300px;
+    }
 
-  &__body {
-    font-size: 19px;
-    text-align: center;
-  }
+    &__title {
+      font-size: 24px;
+      margin: 36px 0;
+      text-align: center;
+    }
 
-  &__button {
-    align-items: flex-end;
-    display: flex;
-    flex: 1;
-    margin-bottom: 98px;
-    width: 100%;
+    &__body {
+      font-size: 19px;
+      text-align: center;
+      margin: 0 36px;
```

Comment:
Maybe you want to define a left/right padding for the whole content div?

Reply:
Yeah, that is good, then I can also remove the padding from the icon. 👍🏽